dojo/index.ts

```diff
@@ -1,49 +1,49 @@
-import * as accordionPane from './accordionPane.m.css';
+import * as accordionPane from './accordion-pane.m.css';
 import * as button from './button.m.css';
 import * as calendar from './calendar.m.css';
 import * as checkbox from './checkbox.m.css';
 import * as comboBox from './comboBox.m.css';
```

Comment:
Any reason we kept this as comboBox?

Reply:
perhaps because i missed it, will look into it